eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -1,146 +1,51 @@
-# Assumes that the VM image is in a variable called OSVmImage
-
 parameters:
   # Use the variable name itself (no $() or other wrapping syntax). This is
   # because we use runtime and macro expressions for conditions and script
   # invocations
   DependenciesVariableName: vcpkg.deps
 
-  # Ref at which to check out (can be commit SHA or tag in the form of
-  # `tags/<tag>`)
-  VcpkgVersion: tags/2020.06
-
 steps:
-  # Set VCPKG_INSTALLATION_ROOT location for cache on Mac OS hosts
-  - task: Bash@3
-    inputs:
-      targetType: inline
-      workingDirectory: $(Agent.TempDirectory)
-      script: |
-        echo "##vso[task.prependpath]$(Agent.TempDirectory)/vcpkg"
-        echo "##vso[task.setvariable variable=VCPKG_INSTALLATION_ROOT;]$(Agent.TempDirectory)/vcpkg"
-    displayName: MacOS - Set VCPKG_INSTALLATION_ROOT
+  - pwsh: |
+      Write-Host "##vso[task.prependpath]$(Agent.TempDirectory)/vcpkg"
+      Write-Host "##vso[task.setvariable variable=VCPKG_INSTALLATION_ROOT]$(Agent.TempDirectory)/vcpkg"
+      Write-Host "##vso[task.setvariable variable=VcpkgRef]$(Get-Content eng/vcpkg-ref.txt)"
+    displayName: Set Vcpkg Variables
```

Comment:
All the console shows is:
```text
Finishing: Set Vcpkg Variables
```

Logging more info about what this step did and some values to console will help diagnosing.